easybuild/easyconfigs/m/maturin/maturin-1.8.3-GCCcore-13.2.0-Rust-1.86.0.eb

```diff
@@ -0,0 +1,756 @@
+easyblock = 'CargoPythonPackage'
+
+name = 'maturin'
+version = '1.8.3'
+_rust_ver = '1.86.0'
```

Comment:
@PetrKralCZ This easyconfig can be removed, no longer needed since we can now combine any Rust version with `maturin` (thanks to #22982)